tracpro/contacts/models.py

```diff
@@ -210,6 +209,9 @@ def _get_first(model_class, temba_uuids):
             'temba_modified_on': temba_contact.modified_on,
             '_data_field_values': temba_contact.fields,  # managed by post-save signal
         }
+        if cls.objects.filter(org=org, uuid=temba_contact.uuid).exists():
+            kwargs['groups'] = [Group.objects.get(uuid=group_uuid) for group_uuid in temba_contact.groups]
```

Comment:
just checking in on the status of this PR @mreartes is it ready to be reviewed again?